mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCDescriptionFragment.java

```diff
@@ -119,15 +169,24 @@ public void onFetchLevelSuccess(KYCLevel1Details kycLevel1Details) {
 
         if (currentLevel >= 1) {
             cvLvl1.setClickable(false);
-            llLvl1.setBackgroundResource(R.drawable.cardview_round_green);
+            text1.setTextColor(getResources().getColor(R.color.colorofkyvlevelsbutton));
```

Comment:
Redundant code repeats

Reply:
@letelete  the llLvl1 had to be changed to white background  no other option

Reply:
You should use a method which handles a proper argument to achieve the same effect but with x3 times less code.

DRY is also really important rule of computer programming. Take a look here:

* https://en.wikipedia.org/wiki/Don%27t_repeat_yourself
or here
* https://code.tutsplus.com/tutorials/3-key-software-principles-you-must-understand--net-25161